kibana-reports/server/model/index.ts

```diff
@@ -27,32 +29,56 @@ import {
 } from '../routes/utils/constants';
 
 export const dataReportSchema = schema.object({
-  base_url: schema.uri(),
+  base_url: schema.string({
+    validate(value) {
```

Comment:
If the validation fails will this still return an error to the client-side and trigger the error toasts in the UI?

Reply:
It will, it will also include the error message